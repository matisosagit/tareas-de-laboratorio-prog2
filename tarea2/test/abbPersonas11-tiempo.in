obtenerExistePersonaABBPersonasTiempo 200000 0.01
Fin
