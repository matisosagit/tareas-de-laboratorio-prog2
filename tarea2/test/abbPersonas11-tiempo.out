1>Se obtuvieron las personas 1 1 1 1 con cedulas respectivas 0 199999 66666 133333
Calculado correctamente en menos de 0.010s.
2>Fin.
